vite.config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  optimizeDeps: {
```

Comment:
Can we remove this `optimizeDeps` block now that we bundling `ipfs-core`?

Reply:
Sadly not, we'd still need this even with the bundled code.